lib/gnat.ex

```diff
@@ -263,6 +343,15 @@ defmodule Gnat do
     %{state | receivers: receivers}
   end
 
+  defp process_message({:msg, topic, @request_sid, reply_to, body}, state) do
+    if Map.has_key?(state.request_receivers, topic) do
+      send state.request_receivers[topic], {:msg, %{topic: topic, body: body, reply_to: reply_to, gnat: self()}}
+      state
+    else
+      Logger.error "#{__MODULE__} got a response for a request, but that is no longer registered"
```

Comment:
Is this going to be annoying?

Reply:
I think we should probably report an error via logger and via the `:error_logger` http://erlang.org/doc/man/error_logger.html. It's part of the standard library and tools like the honeybadger reporter hook into that.

Reply:
@mmmries I wonder if we should just delete this line. If we time out a request and unsub but then later receive that message (ex: db was paused), it will his this code path. Probably doesn't warrant a honey badger style message, or maybe even a log line. If that's the case then we can delete it.

Reply:
Maybe we should just log it with the `debug` level? that way if someone's trying to figure out where their messages are going they can see that timed out requests are showing up later?

Reply:
I would vote `debug` log. That way it definitely can help identify a possible issue if the developer is trying to look for it.